src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/ChainPal.cs

```diff
@@ -110,11 +110,11 @@ public static bool ReleaseSafeX509ChainHandle(IntPtr handle)
                     return;
                 }
 
-                for (int i = 0; i < chainDownloaded.Count; i++)
+                foreach (X509Certificate2 cert in chainDownloaded)
                 {
                     try
                     {
-                        userIntermediate.Add(downloaded[i]);
```

Comment:
@bartonjs, I'm assuming this was a bug, and that this was supposed to be ```chainDownloaded[i]``` rather than ```downloaded[i]```.  Please let me know if I assumed incorrectly.